### PR TITLE
[cloud-provider-yandex](docs) Add to FAQ Adding CloudStatic nodes

### PR DESCRIPTION
**Pull request by unreturned**
## Description

Add to FAQ instruction about Adding CloudStatic nodes to a cluster

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-yandex
type: chore
summary: Add to FAQ instruction about adding CloudStatic nodes.
impact_level: low
```

### CODE REVIEW

File: modules/030-cloud-provider-yandex/docs/FAQ_RU.md [hunk 1]
```diff
@@ -60,3 +60,13 @@ reserved: true
    ```shell
    kubectl edit mc cloud-provider-yandex
    ```
+
+## Добавление CloudStatic узлов в кластер
+
+К виртуальным машинам, которые вы хотите добавить к кластеру в качестве узлов, добавьте в метаданные (Изменить ВМ -> Метадата) ключ `node-network-cidr` с значением равному установленному в кластере `nodeNetworkCIDR`.
```
Comment (z9r5) [inline modules/030-cloud-provider-yandex/docs/FAQ_RU.md:66]:
```suggestion
## Добавление CloudStatic-узлов в кластер

К виртуальным машинам, которые вы хотите добавить к кластеру в качестве узлов, добавьте в метаданные (Изменить ВМ -> Метадата) ключ `node-network-cidr` со значением равному установленному в кластере `nodeNetworkCIDR`.
```

File: modules/030-cloud-provider-yandex/docs/FAQ.md [hunk 2]
```diff
@@ -60,3 +60,13 @@ Do the following to set a custom StorageClass as default:
    ```shell
    kubectl edit mc cloud-provider-yandex
    ```
+
+## Adding CloudStatic nodes to a cluster
+
+For the VMs, you want to add to a cluster as nodes, add in the metadata (Edit VM -> Metadata) key `node-network-cidr` with value equal to the cluster `nodeNetworkCIDR`.
+
+You can find out `nodeNetworkCIDR` using the command:
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ.md:68]:
```suggestion
For VMs that you want to add to the cluster as nodes, add the `node-network-cidr` key to the metadata (Edit VM -> Metadata) with a value equal to the cluster's `nodeNetworkCIDR`.

You can find out the `nodeNetworkCIDR` of the cluster using the command below:
```

File: modules/030-cloud-provider-yandex/docs/FAQ_RU.md [hunk 3]
```diff
@@ -60,3 +60,13 @@ reserved: true
    ```shell
    kubectl edit mc cloud-provider-yandex
    ```
+
+## Добавление CloudStatic-узлов в кластер
+
+К виртуальным машинам, которые вы хотите добавить к кластеру в качестве узлов, добавьте в метаданные (Изменить ВМ -> Метадата) ключ `node-network-cidr` со значением равному установленному в кластере `nodeNetworkCIDR`.
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ_RU.md:66]:
```suggestion
В метаданные виртуальных машин, которые вы хотите включить в кластер в качестве узлов, добавьте (Изменить ВМ -> Метадата) ключ `node-network-cidr` со значением `nodeNetworkCIDR` для кластера.
```

Comment (kirkonru) [reply modules/030-cloud-provider-yandex/docs/FAQ_RU.md:66]: Did some optional rephrase. Imho, it looks better now. Feel free to apply it as you see fit

File: modules/040-node-manager/docs/EXAMPLES_RU.md [hunk 4]
```diff
@@ -84,8 +84,9 @@ spec:
 Чтобы добавить новый статический узел (выделенная ВМ, bare-metal-сервер и т. п.) в кластер вручную, выполните следующие шаги:
 
 1. Для [CloudStatic-узлов](../040-node-manager/cr.html#nodegroup-v1-spec-nodetype) в следующих облачных провайдерах, выполните описанные в документации шаги:
```
Comment (kirkonru) [inline modules/040-node-manager/docs/EXAMPLES_RU.md:86]:
```suggestion
1. Для [CloudStatic-узлов](../040-node-manager/cr.html#nodegroup-v1-spec-nodetype) в облачных провайдерах, перечисленных ниже, выполните описанные в документации шаги:
```

Comment (kirkonru) [reply modules/040-node-manager/docs/EXAMPLES_RU.md:86]: There's always room for improvement 😄

**Review by kirkonru — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.56

**Pull request merged**